apps/src/templates/projects/ProjectWidget.jsx

```diff
@@ -10,6 +13,46 @@ const ProjectWidget = React.createClass({
     projectList: PropTypes.arrayOf(personalProjectDataPropType).isRequired
   },
 
+  statics: {
+    setupProjectWidget() {
+      $.ajax({
+        method: 'GET',
+        url: `/v3/channels`,
+        dataType: 'json'
+      }).done(projectLists => {
+        const widget = document.getElementById('projects-widget');
+        projectLists.sort((a, b) => {
+          if (a.updatedAt < b.updatedAt) {
+            return 1;
+          } else {
+            return -1;
+          }
+        });
+        projectLists = projectLists.filter(project => {
+          return !project.hidden;
+        });
+        projectLists = projectLists.slice(0,4);
+        let convertedProjectList = [];
+        for (let i = 0; i < projectLists.length; i++){
```

Comment:
nit: To some extent personal preference, but I think this would be better expressed as:
```
const convertedProjectList = projectLists.map(projectList => ({
  name: projectList.name,
  ...
  updatedAt: projectList.updatedAt
});
```